scss/partials/portfolio.scss

```diff
@@ -0,0 +1,236 @@
+@use "variables";
+@use "reset";
+
+@import url('https://fonts.googleapis.com/css2?family=Baskervville:ital@0;1&family=Nunito:ital,wght@0,200..1000;1,200..1000&display=swap');
+
+button,
+label {
+    cursor: pointer;
+}
+
+.portfolio {
+    text-align: center;
+    margin-bottom: 42px;
+    .page-header {
+        padding: 46px 0 40px;
+        h2 {
+            font-family: "Baskervville", serif;
+            font-size: 40px;
+            line-height: 40px;
+            text-transform: uppercase;
+            color: variables.$text-darkbrown;
+            position: relative;
+            padding-bottom: 28px;
+            margin-bottom: 28px;
+            font-weight: 400;
+            letter-spacing: 6px;
+            &::after {
+                content: "";
+                display: block;
+                position: absolute;
+                left: 50%;
+                transform: translateX(-50%);
+                bottom: 0;
+                width: 139px;
+                height: 1px;
+                background-color: #E3B6B1;
+            }
+        }
+        p {
+            font-size: 16px;
+            font-family: "Nunito", sans-serif;
+            padding: 0 2rem;
+            font-weight: 200;
+            line-height: 28px;
+        }
+    }
+    .page-tabs {
+        margin-bottom: 24px;
+        width: 100%;
+        max-width: 335px;
+        ul {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            .page-tab {
+                border-bottom: 1px solid #888;
+                position: relative;
+                width: 50%;
+                &::after {
+                    content: "";
+                    display: block;
+                    width: 100%;
+                    height: 1px;
+                    position: absolute;
+                    left: 0;
+                    bottom: -1px;
+                    transition: all .3s ease-in-out;
+                    z-index: 1;
+                }
+                &.on {
+                    .btn-tab {
+                        color: #181818;
+                    }
+                    &::after {
+                        background-color: #181818;
+                    }
+                }
+                // &:last-child.on {
+                //     &::after {
+                //         left: 100%;
+                //     }
+                // }
+            }
+            .btn-tab {
+                color: #888;
+                font-size: 24px;
+                padding: 16px 0;
+                width: 100%;
+                -webkit-appearance: none;
+                appearance: none;
+                background-color: transparent;
+                position: relative;
+                font-family: "Baskervville", serif;
```

Comment:
do you have the reason for not using $primary-font in variables.scss.

※Honestly, I should decide the variable.scss rule first. We now have no uniform rules for the use of VARIABLE files and consider them difficult to use. So, it is my mistake .

Reply:
I wrote that part before adding font variables, but I forgot to update it.
There's another sans-serif font that I'm using in the portfolio.scss file, but it wasn't defined in the variables file.
If you have plan to update variables.scss file later, I will modify after adding it. 
(I wanted to add or change it, but I'm hesitant to modify the reset and variables files. 🥺)

- BTW, I changed font name "Baskervville" to variables.$primary-font. 🙂‍↕️